packages/block/test/testdata/4844-hardfork.ts

```diff
@@ -13,6 +13,7 @@ export const hardfork4844Data = {
     muirGlacierBlock: 0,
     berlinBlock: 0,
     londonBlock: 0,
+    mergeForkBlock: 0,
```

Comment:
This addition should do nothing. If there is one fork at a block, or if there are two forks at the same block: this has no impact on the `forkHash` (note: we should rename this in a different PR to `forkID`) (so both configs will have the same `forkHash`)

Reply:
(but, no need to remove it, since this adds a fork which does nothing)